src/Microsoft.PowerShell.ConsoleHost/host/msh/Telemetry.cs

```diff
@@ -76,7 +67,7 @@ private static void SendTelemetry(string eventName, Dictionary<string,string>pay
             try
             {
                 // if the semaphore file exists, try to send telemetry
-                var enabled = Utils.NativeFileExists(TelemetrySemaphoreFilePath) && !GetEnvironmentVariableAsBool(TelemetryOptoutEnvVar, false);
+                var enabled = GetEnvironmentVariableAsBool(TelemetryOptoutEnvVar, false);
```

Comment:
This is not right. You are returning `false` by default (when the env variable is not set), and thus we will not send telemetry by default. You should pass `true` as the default value.

Reply:
`POWERSHELL_TELEMETRY_OPTOUT = 1` or `POWERSHELL_TELEMETRY_OPTOUT = yes` means to not send telemetry, while `POWERSHELL_TELEMETRY_OPTOUT = 0` or `POWERSHELL_TELEMETRY_OPTOUT = no` means to send telemetry?

This looks so not intuitive to me. When I set `POWERSHELL_TELEMETRY_OPTOUT = no`, I would expect it means no telemetry, literally.

Reply:
My bad, I misread POWERSHELL_TELEMETRY_OPTOUT to POWERSHELL_TELEMETRY_**OUTPUT** ...